lib/media.js

```diff
@@ -36,7 +36,7 @@ exports.init = function () {
           album: item.snippet.description,
           artist: item.snippet.channelTitle,
           artwork: item.snippet.thumbnails.default.url,
-          duration: 0,
+          duration: item.snippet.duration,
```

Comment:
I have checked again and `item` is type of `kind: 'youtube#searchResult'` which does NOT contain duration.